sass/components/_site-switcher.scss

```diff
@@ -16,29 +16,30 @@ Style guide: siteswitcher
   display: block;
   background-color: $grey-lightest;
   font-size: $font-size-little;
+  margin: 0 -15px;
 }
 
 .site-switcher__list {
   @extend .reset-lists;
 }
 
-.site-switcher__option {
+.site-switcher__list li {
   display: inline-block;
   float: left;
   width: 50%;
   text-align: center;
   vertical-align: middle;
+}
 
+li.site-switcher__option {
```

Comment:
I would clean this up to support only the current version, we will change INFO site and co starting to use the latest of the theme.